integration-tests/test-app/sub-title.ts

```diff
@@ -5,7 +5,7 @@ const html = registerHtml();
 /**
  * component to test basic rendering
  */
-const subTitle: TramOneComponent = (props: any, children: Element) => {
```

Comment:
this was previously wrong, children can be a list of elements, not a single element